css-three-column-layout/styles.css

```diff
@@ -0,0 +1,64 @@
+body {
+  background-color: #1a96da;
+  color: white;
+}
+
+h1 {
+  text-align: center;
+  font-family: Lato, sans-serif;
+}
+
+h2 {
+  flex-wrap: wrap;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+p {
+  flex-wrap: wrap;
+  font-family: Lato, sans-serif;
+  font-style: lighter;
+  font-weight: lighter;
+}
+
+img {
+  max-width: 60px;
+  max-height: auto;
+}
+
+.container {
+  max-width: 75%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+  margin: 25px;
+  display: flex;
+  flex-direction: column;
```

Comment:
The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.